tests/MongoDBDriverReadPreference_getMaxStalenessMS-001.phpt

```diff
@@ -1,22 +1,22 @@
 --TEST--
-MongoDB\Driver\ReadPreference::getMaxStalenessMS()
+MongoDB\Driver\ReadPreference::getMaxStalenessSeconds()
 --FILE--
 <?php
 require_once __DIR__ . '/utils.inc';
 
 $tests = [
-    0,
+    90,
```

Comment:
Based on the CDRIVER-1933 code review, I don't believe the read preference struct validates this anymore. The >= 90 validation is deferred until runtime (i.e. server selection). It's probably a good idea to hold off on merging this until their fix is merged.